dev_guide/builds/build_strategies.adoc

```diff
@@ -323,6 +323,27 @@ console], set the *Source Secret*.
 
 . Click the *Save* button.
 
+
+[[docker-strategy-build-args]]
+=== Docker Build Arguments
+
+To set link:http://docs.docker.com/v1.7/reference/api/hub_registry_spec/#docker-registry-1-0[Docker build arguments]
+, add entries to the `BuildArgs` array, which is located in the
```

Comment:
Overall, this looks great! My one nit is that it looks like a weird line break here. Other than that, I think we're ready to go. Thanks!